src/posthog-persistence.ts

```diff
@@ -40,24 +53,17 @@ export class PostHogPersistence {
     cross_subdomain: boolean | undefined
 
     constructor(config: PostHogConfig) {
-        // clean chars that aren't accepted by the http spec for cookie values
-        // https://datatracker.ietf.org/doc/html/rfc2616#section-2.2
-        let token = ''
-
-        if (config['token']) {
-            token = config['token'].replace(/\+/g, 'PL').replace(/\//g, 'SL').replace(/=/g, 'EQ')
-        }
-
+        this.config = config
         this.props = {}
         this.campaign_params_saved = false
-        this.custom_campaign_params = config['custom_campaign_params'] || []
```

Comment:
lost `this.custom_campaign_params = config['custom_campaign_params'] || []` on purpose?

Reply:
We have the whole config now so we just reference it directly where it is used.